.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
```

Comment:
What if its a feature request?

Reply:
@bronzdoc There is already a template for feature requests in this project. 👍 

When submitting a new issue, they will be given a choice to select one of these options: 

- Bug Report 
- Feature Request 

See: https://github.com/fastruby/skunk/blob/main/.github/ISSUE_TEMPLATE/feature_request.md